### PR TITLE
Implement Vendor Specific Instructions property for site generation

### PR DESCRIPTION
**Pull request by tarilabs**
... with an actual example for Drools result, as agreed during last DMN
TCK working group call (06-Apr-2018)

This is a proposal for the implementation.
Actual "wordings" on the website to be discussed/agreed on.
The instruction URL is currently referenced in two places:
image

and in the details pages, along with the pre-existing disclaimer:
image

Please notice if the URL of the instruction is not present in the results' property file, simply this new section is not rendered (instead of rendering "no instruction available").

/cc @etirelli can you kindly review also specifically the Drools' runner instruction manual for me, please?

### CODE REVIEW

File: TestResults/Drools/7.6.0.Final/tck_results.properties [hunk 1]
```diff
@@ -5,3 +5,4 @@ product.url=http://www.drools.org
 product.version=7.6.0.Final
 product.comment=Drools provides a compliance level 3 runtime DMN engine
 last.update=2018-02-22
+tck.runner.manual.url=https://github.com/dmn-tck/tck/tree/master/runners/dmn-tck-runner-drools/README.md
```
Comment (etirelli) [inline TestResults/Drools/7.6.0.Final/tck_results.properties]: nitpick: I would simplify the property to "instructions.url". not all vendors might provide a runner, and instead just use some kind of tooling.

File: runners/dmn-tck-reporter/src/main/java/org/omg/dmn/tck/Vendor.java [hunk 2]
```diff
@@ -28,6 +28,7 @@
     private String                  comment;
     private Map<String, TestResult> results;
     private LocalDate               lastUpdate;
+    private String                  tckRunnerManualUrl;
```
Comment (etirelli) [inline runners/dmn-tck-reporter/src/main/java/org/omg/dmn/tck/Vendor.java]: and just for consistency, I would rename the attribute as well.

File: runners/dmn-tck-reporter/src/main/resources/templates/index.ftl [hunk 3]
```diff
@@ -196,6 +196,9 @@
                                     <dt>Failed: </dt><dd>${vendor.failed}</dd>
                                     <dt>Skipped: </dt><dd>${header.totalTests - vendor.succeeded - vendor.failed}</dd>
                                     <dt>Info: </dt><dd>${vendor.comment}</dd>
+                        		<#if vendor.tckRunnerManualUrl?has_content>
+									<dt>TCK Runner manual: </dt><dd>This vendor has supplied an instruction manual for user to reproduce TCK test results, available <a href="${vendor.tckRunnerManualUrl}" target="_blank">at this link <span class="fa fa-external-link"></span></a>.</dd>
+								</#if>
```
Comment (etirelli) [inline runners/dmn-tck-reporter/src/main/resources/templates/index.ftl]: Suggestion for wording: 

* if the instruction URL is available:

> Instructions: to execute the tests and reproduce the results, please follow the vendor provided instructions at this link.

* if instructions are not provided:

> Instructions: this vendor did not provide instructions to execute the tests. Please contact the vendor directly.

**Comment by etirelli** [comment]
@tarilabs other than my nickpicking comments, looks good, thank you!

**Comment by tarilabs** [comment]
Applying code review comments results in:
image

and on each Vendor's page in:
image

or:
image

**Pull request merged**